sql/select.go

```diff
@@ -204,9 +204,10 @@ func (p *planner) selectIndex(s *scanNode, ordering []int) (planNode, error) {
 	sort.Sort(indexInfoByCost(candidates))
 
 	if log.V(2) {
+		// TODO(pmattis): Show all of the constraints?
```

Comment:
I'm going to work on this next in a separate PR along with additional testing of `make{Constraints,Spans}`.